### DISCUSSION TITLE
Single kitty config across multiple machines

### DISCUSSION THREAD

**Discussion by ssbanerje** [kovidgoyal/kitty#4122] [Q&A] [answered]
Hi,
Is there a way to enable parts of the kitty config conditionally? 

I am looking for some combination of `if` like conditionals and variables. I would like to use the same kitty file across my Linux and Mac machines and want to set up my key remaps like so (this doesn't work, is python-like pseudocode):
```
MOD_KEY = cmd if ($(uname -s) == Darwin) else ctrl
map MOD_KEY+c copy_to_clipboard
``` 
The default config file and the docs point to the `map` keyword, but not the rest.

Maybe even some combination of a conditional and an `include` might also achieve the same behavior.

**Comment by Luflosi** [comment, marked as answer]
See https://github.com/kovidgoyal/kitty/issues/811#issuecomment-413896659 and https://github.com/kovidgoyal/kitty/issues/811#issuecomment-414186903 for two alternatives that will allow you to do what you want.
Use something like `kitty_mod super` on the Mac and `kitty_mod ctrl+shift` on Linux and also define `map kitty_mod+c copy_to_clipboard`.
